tutorials/io/data_paths.rst

```diff
@@ -37,8 +37,8 @@ when the game is running, the project's file system will likely be read-only.
 
 The ``user://`` prefix points to a different directory on the user's device. On
 mobile and consoles, this path is unique to the project. On desktop, the engine
-stores user files in ``~/.local/share/godot/app_userdata/Name`` on macOS and
-Linux, and ``%APPDATA%/Name`` on Windows. ``Name`` is based on the application
+stores user files in ``~/.local/share/godot/app_userdata/Name`` on
+Linux, ``~/Library/Application Support/Godot/app_userdata`` on macOS (since Catalina) and ``%APPDATA%/Name`` on Windows. ``Name`` is based on the application
```

Comment:
```suggestion
Linux, ``~/Library/Application Support/Godot/app_userdata/Name`` on macOS (since Catalina) and ``%APPDATA%/Name`` on Windows. ``Name`` is based on the application
```